.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: NodeJS with Webpack
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
```

Comment:
can we do 17.x as well? 16.x is the latest LTS, but 17.x is the "current" node version

Reply:
Sure, done!